docs/docs/about/release-notes.mdx

```diff
@@ -85,6 +85,7 @@ sidebar_position: 4
 - removed Node 16 workflow
 - aligned options and documentation power platform. [#5411](https://github.com/pnp/cli-microsoft365/issues/5411)
 - renamed option for 'spo user get'. [#5515](https://github.com/pnp/cli-microsoft365/issues/5515)
+- renamed 'search externalconnection [verb]' commands to 'to external connection [verb]' [#5531](https://github.com/pnp/cli-microsoft365/issues/5531)
```

Comment:
Let's remove this, `v7.0` is already released. A maintainer will add this to the release notes under the right version. If contributors update the release notes, it will cause a tonne of merge conflicts.